### ISSUE TITLE
'elif obj.is_comment and (obj.indent == 0)' does not process banner if it contains '!'

### ISSUE THREAD

**Issue by mynameisosama** [mpenning/ciscoconfparse#115]
The following:
`
elif obj.is_comment and (obj.indent == 0):
    break
`
in `def _banner_mark_regex(self, REGEX)`, breaks the loop and does not process the whole banner if it contains '!' at the start of new line. Is there a particular reason for this condition?

example config:
banner exec ^CCC
              !!!!!SITIO  RESTRINGIDO!!!!!
!!!!!ACCESO PERMITIDO SOLO A PERSONAL AUTORIZADO!!!!!
 ^C

**Comment by mpenning** [comment]
I think this is fixed in the latest version of CiscoConfParse... could you download and check?

```
>>> from ciscoconfparse import CiscoConfParse
>>> parse = CiscoConfParse('foo.conf')
>>> parse.find_objects(r'^banner')
[<IOSCfgLine # 3 'banner exec ^CCC'>]
>>> bobj = parse.find_objects(r'^banner')[0]
>>> bobj.children
[<IOSCfgLine # 4 '!!!!!SITIO RESTRINGIDO!!!!!' (parent is # 3)>, <IOSCfgLine # 5 '!!!!!ACCESO PERMITIDO SOLO A PERSONAL AUTORIZADO!!!!!' (parent is # 3)>, <IOSCfgLine # 6 '^C' (parent is # 3)>]
>>>
```

**Comment by mpenning** [comment]
This was fixed in version 1.3.21
